test/evergreen.yml

```diff
@@ -40,6 +40,25 @@ functions:
       params:
         path: "WT_TEST-big-endian.tgz"
         destination: "wiredtiger/build_posix/test/format"
+  "fetch artifacts from many-coll" :
+    - command: s3.get
+      params:
+        aws_key: ${aws_key}
+        aws_secret: ${aws_secret}
+        remote_file: dbpath_100k_500.tgz
```

Comment:
Using a hardcoded filename means a new run of the task (calling this function) will overwrite the artifacts file of an old run. And that's why we have `${build_id}` as part of the artifacts filename in the common `fetch artifacts` function. Also suggest to store the artifacts in `wiredtiger/${build_variant}/${revision}/artifacts/` directory structure to be conform with other wiredtiger tasks. Basically, use the common `fetch artifacts` if possible.

Reply:
@lukech, we have uploaded a database that we don't want to overwrite anymore but we just want to fetch it before the start of the many-collection-test. I am not sure we can use the common `fetch artefacts` method.

Reply:
I see, in your case the artifacts is more of a static file - not dynamically generated.